packages/stark-core/assets/css/pre-load-style.css

```diff
@@ -0,0 +1,93 @@
+/* Hide the app / loader / browser warning until it is determined if JS is enabled and the if the browser is compatible  */
+.stark-app,
+.stark-loading,
+.stark-browser-warning {
+  display: none;
+}
+
+.stark-browser-outdated .stark-browser-warning {
+  display: block;
+}
+
+.stark-browser-ok .stark-app {
+  display: flex;
+}
+
+.stark-browser-ok .stark-loading {
+  display: block;
+}
+
+.stark-preload-ui-page h1 {
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 32px;
+  margin: 0 0 16px;
+}
+
+.stark-preload-ui-page {
+  color: #000;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue,
+    sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 26px;
+
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  box-sizing: border-box;
+  max-width: 100%;
+  width: 500px;
+  overflow: hidden;
+  border: solid 1px rgba(0, 0, 0, 0.12);
+  box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12);
+
+  margin: 16px auto;
+  border-radius: 8px;
+  text-align: center;
+}
+
+.stark-preload-ui-page header {
+  padding: 16px;
+  background: #143e74 url("/assets/stark-core/logo/stark_header_bg.png") top right no-repeat;
+  border-radius: 2px 2px 0 0;
+}
+
+.stark-preload-ui-page header i {
+  margin: auto;
+  display: block;
+  width: 200px;
+  height: 60px;
+  background: url("/assets/stark-core/logo/stark_logo_transparent.svg") no-repeat;
+  background-size: 100% 100%;
+}
+
+.stark-preload-ui-page .content {
+  display: block;
+  padding: 16px;
+  background: #fff;
+}
+
+.stark-preload-ui-page .stark-loading-icon {
+  margin: 0 auto 8px auto;
+  display: block;
+  width: 64px;
+  height: 64px;
+  background: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI2NCIgaGVpZ2h0PSI2NCIgeD0iMCIgeT0iMCIgdmlld0JveD0iMCAwIDUwIDUwIj48cGF0aCBmaWxsPSIjN2ZiYWUzIiBkPSJNMjUuMjUxLDYuNDYxYy0xMC4zMTgsMC0xOC42ODMsOC4zNjUtMTguNjgzLDE4LjY4M2g0LjA2OGMwLTguMDcxLDYuNTQzLTE0LjYxNSwxNC42MTUtMTQuNjE1VjYuNDYxeiI+PC9wYXRoPjwvc3ZnPg==");
```

Comment:
I think the exact same CSS for the `.stark-loading-icon` is also defined in `packages/stark-ui/assets/styles/_base.scss`. So I suggest to remove that one in Stark-UI in favor of this one ;)

Reply:
The problem would be that when a user decides to not use the css the `<stark-preloading-page>` component would not have the animation anymore.

Reply:
Ah true! I completely forgot about that! So, let's keep it then ;)